analyze-wasm/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@arcjet/analyze-wasm",
+  "version": "1.0.0-alpha.34",
+  "description": "WebAssembly bindings to Arcjet's local analysis engine",
+  "license": "Apache-2.0",
+  "homepage": "https://arcjet.com",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arcjet/arcjet-js.git",
+    "directory": "analyze-wasm"
+  },
+  "bugs": {
+    "url": "https://github.com/arcjet/arcjet-js/issues",
+    "email": "support@arcjet.com"
+  },
+  "author": {
+    "name": "Arcjet",
+    "email": "support@arcjet.com",
+    "url": "https://arcjet.com"
+  },
+  "engines": {
+    "node": ">=18"
+  },
+  "type": "module",
+  "main": "./index.js",
+  "types": "./index.d.ts",
+  "exports": {
+    "edge-light": "./edge-light.js",
+    "workerd": "./workerd.js",
+    "default": "./index.js"
+  },
+  "files": [
+    "LICENSE",
+    "README.md",
+    "_virtual/",
+    "wasm/",
+    "*.js",
+    "*.d.ts",
+    "*.ts",
+    "!*.config.js"
+  ],
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "jco": "jco transpile wasm/arcjet_analyze_js_req.component.wasm --no-wasi-shim --instantiation async -o wasm",
+    "build": "npm run jco; rollup --config rollup.config.js",
+    "lint": "eslint .",
+    "pretest": "npm run build",
+    "test": "node --test"
+  },
+  "devDependencies": {
```

```suggestion
  "dependencies": {},
  "devDependencies": {
```
Nit, I keep the dependencies field around even without any so it is easier to add them later.